epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -88,21 +87,30 @@ <h2>How EPUB conforming to EPUB accessibility guidelines cover European Accessib
 
 			<ul>
 				<li>discoverability of accessibility features of publications;</li>
-				<li>evaluation and certification of accessible EPUB publications.</li>
+				<li>conformance to Web Content Accessibility Guidelines;</li>
```

Comment:
(I cannot edit directly) the preceding text says 

> address two key needs in the EPUB ecosystem:

I guess this now should say "four key need", or possible "the following key needs"